# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/alanw/afio/notusrlocal.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Naur afio-2.5.orig/Makefile afio-2.5/Makefile
--- afio-2.5.orig/Makefile	2003-12-21 10:23:51.000000000 +0000
+++ afio-2.5/Makefile	2005-01-26 09:44:00.000000000 +0000
@@ -88,7 +88,7 @@
 	cd regtest; /bin/sh regtest.clean
 
 install: afio
-	cp afio /usr/local/bin
+	cp afio /usr/bin
 	cp afio.1 /usr/share/man/man1
 
 # generate default list of -E extensions from manpage
